### PR TITLE
SAK-29489: Use Fonts (FontAwsome) for icons replacing images

### PR DESCRIPTION
**Pull request by udayg**
Use Fonts (FontAwsome) for icons replacing images.

Fixing CSS for profile image

### CODE REVIEW

**Comment by ottenhoff** [comment]
Awesome! +1 thanks Uday

File: reference/library/src/morpheus-master/sass/modules/tool/_base.scss [hunk 1]
```diff
@@ -29,23 +29,36 @@
...
 		&--edit{
-			@include base-sprite(edit);	
+			@extend .fa;
+			@extend .fa-pencil;
+			@extend .fa-lg;	
 		}
 		&--reset{
-			@include base-sprite(reload);	
+			@extend .fa;
+			@extend .fa-home;
```
Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/modules/tool/_base.scss]: Maybe it should extend .fa-refresh, what do you think?

Comment (udayg) [inline reference/library/src/morpheus-master/sass/modules/tool/_base.scss]: We can use that but, there was a debate on what to use here, Sakai for long time used the refresh button but the actual function of this button is more of take me to the portal/tool home than refresh. When Mark did usability study in NYU he has done this change to make it as home in his initial design, I personally agree with his idea too, but we can discuss this during  next Thursday feedback session to take more input on what it should be by default.

Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/modules/tool/_base.scss]: I was doubting if users would be comfortable with this change but if it is tested yet it's good to me :)

**Pull request merged**